src/librustdoc/html/static/js/main.js

```diff
@@ -1392,10 +1392,10 @@ href="https://doc.rust-lang.org/${channel}/rustdoc/read-documentation/search.htm
              <code>-&gt; vec</code> or <code>String, enum:Cow -&gt; bool</code>)",
             "You can look for items with an exact name by putting double quotes around \
              your request: <code>\"string\"</code>",
-             "Look for functions that accept or return \
-              <a href=\"https://doc.rust-lang.org/std/primitive.slice.html\">slices</a> and \
-              <a href=\"https://doc.rust-lang.org/std/primitive.array.html\">arrays</a> by writing \
-              square brackets (e.g., <code>-&gt; [u8]</code> or <code>[] -&gt; Option</code>)",
+             `Look for functions that accept or return \
+              <a href=\"${drloChannel}/std/primitive.slice.html\">slices</a> and \
```

Comment:
The CI error is talking about cases like there. Before it used `"` for the string, so the literal `"` in the HTML had to be escaped. Now you're using ``` ` ``` for the string, so the literal `"` does not need escaping -- and the linter doesn't like that it still is escaped.

There are several other cases elsewhere in this file and the other too.